README.md

```diff
@@ -24,59 +24,30 @@ found in the `Makefile` into your command line interface.
  by unit tests
 
 ## Adding and removing dependencies
+This project uses [Poetry](https://poetry.eustace.io/) for dependency management.
 
-The following commands update files that keep track of project dependencies.
-
-```bash
-make add-dependency package=<package name>
-make add-dev-dependency package=<package name>
-
-# examples:
-make add-dependency package=flask
-make add-dependency package=flask==1.0.2
-make add-dev-dependency package=pytest
-```
-These commands add a python package as a project dependency or 
-development dependency alike. You can specify just the package name to get the 
-latest version or specify a specific version number.
-
-```bash
-make remove-dependency package=<package name>
-make remove-dev-dependency package=<package name>
-```
-These commands remove dependencies accordingly.
+To [add](https://poetry.eustace.io/docs/cli/#add) and [remove](https://poetry.eustace.io/docs/cli/#remove)
+a project dependency, run `make shell` to enter the bash shell of the app container
+and make changes as necessary. Any changes made will only update `pyproject.toml`
+and `poetry.lock` files. If you would like to apply the changes to the app container
+then rebuild the container by running `make build`.
 
+Example:
 ```bash
+make shell
+poetry add pytest --dev
```

Comment:
sometimes it's useful to show the prompts here to show the change of context:
```
$ make shell
bash-4.4# ls search/
```
Though it becomes more a sample output than a list of commands.

Reply:
I did think about adding the prompt. As it's an example . I think I will.

Reply:
shell added.